.eslintrc.json

```diff
@@ -0,0 +1,42 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "cypress/globals": true
+  },
+  "globals": {
+    "global": "readonly",
+    "jest": "readonly"
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:prettier/recommended",
+    "plugin:cypress/recommended"
+  ],
+  "plugins": ["jest", "cypress"],
```

Comment:
This doesn't need to live in the root level configuration if you have it in the overrides